webid.go

```diff
@@ -15,42 +15,37 @@ import (
 	"time"
 )
 
-var (
-	agentProfile   string
-	webidTlsClient *http.Client
-	agentCert      *tls.Certificate
-	privKey        *rsa.PrivateKey
-
-	err error
+type Agent struct {
+	WebID   string
+	Profile string
+	Cert    *tls.Certificate
+	Key     *rsa.PrivateKey
+}
 
+var (
+	err            error
 	subjectAltName = []int{2, 5, 29, 17}
```

Comment:
`Subject Alternative Name` is the field that holds the WebID URI in an x509 client certificate.